editor/animation_track_editor.cpp

```diff
@@ -306,8 +306,9 @@ bool AnimationTrackKeyEdit::_set(const StringName &p_name, const Variant &p_valu
 				setting = true;
 				undo_redo->create_action(TTR("Animation Change Keyframe Value"), UndoRedo::MERGE_ENDS);
 				int prev = animation->bezier_track_get_key_handle_mode(track, key);
-				undo_redo->add_do_method(this, "_bezier_track_set_key_handle_mode", animation.ptr(), track, key, value);
-				undo_redo->add_undo_method(this, "_bezier_track_set_key_handle_mode", animation.ptr(), track, key, prev);
+				ERR_FAIL_COND_V(!editor, false);
```

Comment:
```suggestion
				ERR_FAIL_NULL_V(editor, false);
```
Use proper error check